src/barbequeue/storage/backends/default.py

```diff
@@ -1,4 +1,8 @@
 import abc
+from collections import defaultdict
+from threading import Event
+
+JOB_EVENT_MAPPING = defaultdict(lambda: Event())
```

Comment:
Documentation: _dict subclass that calls a factory function to supply missing values_

https://docs.python.org/2/library/collections.html

Since there's a difference between Python 2 and 3, you will have a call to a constructor in Python 2 and this in Py3:

``` 
>>> from threading import Event
>>> Event()()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'Event' object is not callable
```